documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
+Search on [org-id.guide](http://org-id.guide) for identifier sources for [UK organisations](http://org-id.guide/?structure=&coverage=GB&subnational=&sector=), [UK charities](http://org-id.guide/?structure=charity&coverage=GB&sector=), or [any other organisation type](http://org-id.guide/).
 
-5. **No registered number**
+### Commonly used identifier lists
 
-    If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier* you have for this organisation.
-    
-    *If you use a database that records details of organisations in a separate lookup table, this may provide an identifier you can use. 
+The following identifier lists are often used in 360Giving publication. They are listed here in rough order of priority (e.g. if you already know the company number, use this in preference to the charity number).
+
+* [GB-COH](http://org-id.guide/list/GB-COH) - UK Company Number
+* [GB-CHC](http://org-id.guide/list/GB-CHC), [GB-SC](http://org-id.guide/list/GB-SC), [GB-NIC](http://org-id.guide/list/GB-NIC) - Charity Numbers
+* [GB-EDU](http://org-id.guide/list/GB-EDU) and [GB-UKPRN](http://org-id.guide/list/GB-UKPRN) - Education establishments
+* [GB-LAE](http://org-id.guide/list/GB-LAE) (England), [GB-LAS](http://org-id.guide/list/GB-LAS) (Scotland), [GB-PLA](http://org-id.guide/list/GB-PLA) (Wales)  - Local authorities
+
+
+If you have a registered number from some other scheme, including overseas registrars, check the [org-id List Locator](http://org-id.guide/) for a Registration Agency Code to use. If the Registration Agency Code you need is not listed, [contact the support team](http://www.threesixtygiving.org/contact/).
+
+If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier you have for this organisation.
     
-    If you only record data in a spreadsheet, and don't assign organisations an ID, you could use a spreadsheet formula to turn the organisation name into an identifier (e.g. removing spaces and lowercasing the name). The support team can provide guidance on this. 
+If you use a database that records details of organisations in a separate lookup table, this may provide an identifier you can use. 
     
-    If there is a chance that your organisation identifiers might overlap with grant identifiers, just add 'ORG' into the identifier string (e.g. '360G-xyztrust-ORG123')
+If you only record data in a spreadsheet, and don't assign organisations an ID, you could use a spreadsheet formula to turn the organisation name into an identifier (e.g. removing spaces and lowercasing the name). The support team can provide guidance on this. 
```

Comment:
Again, not sure this is the best way going forward. I think we do want them to record the numbers when they exist. It's more work, I am aware, but I do think our publishers start to understand the importance of this

Reply:
See my comment above. I'd strongly prefer we encourage them to use established identifiers.